ci-operator/config/openshift/assisted-image-service/openshift-assisted-image-service-release-ocm-2.4.yaml

```diff
@@ -93,6 +117,14 @@ tests:
     make test
   container:
     from: test-bin
+- as: e2e-metal-assisted
```

Comment:
not sure we need to handle 2.4 branch
we'll also need to backport your changes in assisted-service to 2.4 branch for that

Reply:
Agree, removed all 2.4 changes